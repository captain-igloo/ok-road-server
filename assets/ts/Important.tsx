import * as React from 'react';

interface Props {
    children: React.ReactNode;
}

export default function Important(props: Props) {
    const { children } = props;

    return (
        <div className="alert__important">
            <p className="alert__important--title">
                <img alt="" src="/img/important.svg" width={16} />
                {' '}
                Important
            </p>
            <p className="alert__important--body">
                {children}
            </p>
        </div>
    );
}
