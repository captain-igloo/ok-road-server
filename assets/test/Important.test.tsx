import { describe, expect, test } from '@jest/globals';
import { render } from '@testing-library/react';
import * as React from 'react';

import Important from '../ts/Important';

describe('Important component', () => {
    test('Important renders properly', () => {
        const { container } = render(<Important>Message!</Important>);
        expect(container).toMatchSnapshot();
    });
});
