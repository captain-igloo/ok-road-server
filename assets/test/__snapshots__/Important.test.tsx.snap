// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Important component Important renders properly 1`] = `
<div>
  <div
    class="alert__important"
  >
    <p
      class="alert__important--title"
    >
      <img
        alt=""
        src="/img/important.svg"
        width="16"
      />
       
      Important
    </p>
    <p
      class="alert__important--body"
    >
      Message!
    </p>
  </div>
</div>
`;
